src/platform/Kconfig

```diff
@@ -530,4 +530,10 @@ config FORCE_DMA_COPY_WHOLE_BLOCK
 	  The host component forces DMA to copy the block size to avoid
 	  copying byte jitter between the components of the same pipeline.
 
+config SOF_STACK_SIZE
+	int "Primary core SOF stack size"
+	default 4096
+	help
+	  Size of the primary core stack.
```

Comment:
Can we have a description of why would would want to change this. Fwiw, I think this is already an option with Zephyr.